graylog2-server/src/main/java/org/graylog/security/shares/GranteeService.java

```diff
@@ -24,6 +24,8 @@
 public interface GranteeService {
     Set<Grantee> getAvailableGrantees(User sharingUser);
 
+    Set<Grantee> getSelectableGrantees(Set<Grantee> availableGrantees);
```

Comment:
maybe rename it to modifiableGrantees which the user can modify, available to get all of them to be able to get activeShares in the [EntitySharesService.java](https://github.com/Graylog2/graylog2-server/pull/12797/files#diff-ba8058310df814ed5be1c2188dbe77ab670f9758bf080eee4689e41bdc10f3b8)